eslint/cactbot-matches-type-references.js

```diff
@@ -0,0 +1,72 @@
+const findTriggerIdFromSiblingProperty = (node) => {
+  return node.parent.parent.properties.find((property) => property.key.name === 'id').value.value;
```

Comment:
This is...quite a statement.  I guess there's no way to make this simpler or more readable?

Reply:
I could spend the time to say what it's doing, but not really? We're inside of a CallExpression, so you have to go up to get to the Property, and then up once more to get to the whole trigger ObjectLiteral. From there, we find the id and then you have to get the value's value, which is to say the actual content inside the quoted trigger id.

I can make that more explicit here if we feel like that's necessary or useful, sure